components/src/main/kotlin/com/mineinabyss/geary/ecs/helper/ComponentsHelper.kt

```diff
@@ -0,0 +1,9 @@
+package com.mineinabyss.geary.ecs.helper
+
+import com.mineinabyss.geary.ecs.api.GearyComponent
+import com.mineinabyss.geary.ecs.serialization.Formats
+import kotlin.reflect.KClass
+
+public class Components (names: Collection<String>) {
```

Comment:
I think `Components` is a bit too broad of a name. Also no space between class and `(`. (I always spam `Ctrl + Alt + L` to do auto formatting, but you can also turn it on as an automatic step before every commit in intellij :p)

Reply:
I don't think we need a class here anymore until we have at least 2 different props inside